src/test/java/org/qafellas/tests/StringTests.java

```diff
@@ -29,4 +29,10 @@ public void reverseString(){
         Assert.assertEquals(sf.reverseString(str), "pac");
     }
 
+    @Test
+    public void stringContainsLetter(){
+        boolean status = sf.contains("New york", "k");
```

Comment:
Hey there is a typo in "New york" fix it

Reply:
I fixed